
Not a new checkout, because the upstream netconf exact-build migration is not yet finished.

--- ./gui/ppp_device.hh	(revision 285)
+++ ./gui/ppp_device.hh	(revision 286)
@@ -32,7 +32,7 @@
   void selectionChanged (int item); // by ComboBox
 
 private:
-  void PPPDevice::SetPPPOE (bool value); // by refreshContent()
+  void SetPPPOE (bool value); // by refreshContent()
   void SwitchType ();  // common stuff by selectionChanged / SetPPPOE
 
   QString title;
